core/frontend/src/components/bridges/BridgeCreationDialog.vue

```diff
@@ -161,6 +161,17 @@ export default Vue.extend({
       return system_serial_ports
         .filter((serial_info) => bridget.available_serial_ports.includes(serial_info.name))
     },
+    bridge_mode(): string {
+      if (['0.0.0.0', '127.0.0.1'].includes(this.bridge.ip)) {
```

Comment:
A question: if using `127.0.0.1`, bridges will only accept connections from devices in the same machine or it will accept requests from other devices within the network?

Reply:
> A question: if using `127.0.0.1`, bridges will only accept connections from devices in the same machine or it will accept requests from other devices within the network?

Same machine, it's still a valid serve mode and the user may want to create something for intern usage, but it's not our default (that still is `0.0.0.0`).

Reply:
From the definition of 127.0.0.1 I would expect it to only accept connections from localhost

Reply:
It's the same understand I have. So maybe we should differentiate between both (maybe with something like "local only server" on the `127.0.0.1`)?

Reply:
Sounds like a good idea. For sure someone would try to use localhost with external stuff trying to reach it

Reply:
I agree

Reply:
@rafaellehmkuhl check now